frappe/workflow/doctype/workflow/test_workflow.py

```diff
@@ -93,6 +94,40 @@ def test_if_workflow_actions_were_processed(self):
 		self.assertEqual(workflow_actions[0].status, 'Completed')
 		frappe.set_user('Administrator')
 
+	def test_if_workflow_actions_were_processed_using_user(self):
+		frappe.db.delete("Workflow Action")
+		if not frappe.db.has_column('Workflow Action', 'user'):
+			# mariadb would raise this state would create an implicit commit
+			# if we do not commit before alter statement
+			# nosemgrep
+			frappe.db.commit()
+			frappe.db.multisql({
+				'mariadb': 'ALTER TABLE `tabWorkflow Action` ADD COLUMN user varchar(140)',
+				'postgres': 'ALTER TABLE "tabWorkflow Action" ADD COLUMN "user" varchar(140)'
+			})
```

Comment:
PS: We could maintain these under the setUp method alone too, with an `if self._testMethodName == "test_if_workflow_actions_were_processed_using_user"`

Reply:
DOne